src/codegen/llvm/target_platform.hpp

```diff
@@ -32,11 +32,16 @@ class Platform {
     /// Name of the platform.
     const std::string name = Platform::DEFAULT_PLATFORM_NAME;
 
+    /// Target chip for GPUs.
+    /// TODO: this should only be available to GPUs! If we refactor target
+    /// classes so that GPUPlatform <: Platform, it will be nicer!
+    const std::string subtarget_name = "sm_35";
```

Comment:
@iomaganaris : considering we are now mostly running stuff on Volta GPUs, it's better to have this `sm_70`? (could be `sm_60` for Pascal generation).

Reply:
Yes, I think `sm_70` makes sense since most HPC and local computer systems now use GPU with Compute Capability 7+